react-elements/src/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import * as ReactDOM from 'react-dom';
```

Comment:
This is how the dcos show the import statement for `ReactDOM`, but you can shorten it to:
```JAVASCRIPT
import ReactDOM from 'react-dom/client';
```